components/Footer.vue

```diff
@@ -5,6 +5,11 @@
       in Vancouver, Canada. Learn something new at UBC Local Hack Day, make new friends across the Pacific Northwest
       and around the globe at nwHacks, and join an empowering community of women* in tech at cmd-f.
     </p>
+    <div class="tv">
```

Comment:
Not sure if it's just my screen but the text leaves the box.
![image](https://user-images.githubusercontent.com/13321051/99915319-9cc26900-2cc8-11eb-9a78-c023a24e4168.png)

Reply:
Should this just be part of the graphic then?

Reply:
That's one possible option. I was wondering if this box for displaying the team should be its own component. That way the text would always be contained inside the box. This would also help when adding all the images and making it horizontally scrollable.